test/conformance/chainsaw/verifyImages/clusterpolicy/standard/sigstore-attestation-verification-regexp/pod.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+  namespace: default
+spec:
+  containers:
+  - image: ghcr.io/vishal-chdhry/artifact-attestation-example:artifact-attestation
```

Comment:
Can we test an image from Kyverno repository?

Reply:
this was signed by a workflow and I cannot give the repo access to an image in Kyverno org